tools/hsmtool.c

```diff
@@ -417,6 +419,25 @@ static int dumponchaindescriptors(const char *hsm_secret_path, const char *passw
 
 	wally_free_string(enc_xpub);
 
+	/* If they also want xpriv break a line and output them as well. */
+	if (private) {
+		printf("\n");
+		if (bip32_key_to_base58(&master_extkey, BIP32_FLAG_KEY_PRIVATE, &enc_xpub) != WALLY_OK)
```

Comment:
you're writing the `enc_xpriv` to a variable named `enc_xpub`

Reply:
This commit is now removed.